include/mips/pmap.h

```diff
@@ -1,10 +1,6 @@
 #ifndef _MIPS_PMAP_H_
 #define _MIPS_PMAP_H_
 
-#ifndef _MACHDEP
-#error "Do not use this header file outside kernel machine dependent code!"
-#endif
-
```

Comment:
A I right that you can now restore this lines?

Reply:
restored